tests/spec/fs.rename.spec.js

```diff
@@ -70,6 +70,33 @@ describe('fs.rename', function() {
     });
   });
 
+  it('should rename an existing directory (using promises)', () => {
+    var fsPromises = util.fs().promises;
+
+    return fsPromises.mkdir('/mydir')
+      .then(() => {
+        return fsPromises.rename('/mydir', '/myotherdir')
+          .catch((error) => expect(error).not.to.exist);
```

Comment:
Logical error here: By catching an error, you would expect an error to already exist
You could just remove this line